libs/legi-shared/list-paginator/list-paginator.component.ts

```diff
@@ -17,17 +20,26 @@ export class ListPaginatorComponent extends OnDestroy$ implements OnInit, OnChan
   @Output()
   changePagination: EventEmitter<Pagination> = new EventEmitter();
 
+  /**
+   *  `sizes` property determines how many items per page
```

Comment:
```suggestion
   * property determines how many items should be displayed per page
```
In general, there is no need to repeat the name of whatever is being commented on, please use `It`, `This` or something similar instead (also tru for classes and methods documenting)